<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HDF5Explorer_Widget</class>
 <widget class="QWidget" name="HDF5Explorer_Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HDF5 Explorer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>HDF5 File Explorer</string>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="fileSelectLayout">
     <item>
      <widget class="QLineEdit" name="filePathEdit">
       <property name="placeholderText">
        <string>Select an HDF5 file to browse...</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="browseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
       <property name="toolTip">
        <string>Select an HDF5 file to open</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh</string>
       </property>
       <property name="toolTip">
        <string>Reload the current file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
     </property>
     <property name="headerHidden">
      <bool>false</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dimensions</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="infoGroupBox">
     <property name="title">
      <string>Selection Info</string>
     </property>
     <layout class="QVBoxLayout" name="infoLayout">
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string>Select a file to browse its structure</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::RichText</enum>
        </property>
        <property name="minimumHeight">
         <number>80</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
